src/index.js

```diff
@@ -126,15 +130,15 @@ if (process.env.NODE_ENV !== 'production') {
 		...appParams,
 		...integrationTestOverrides,
 	};
-	initialize(dictSettings);
+	window.DrugDictionaryApp(dictSettings);
 } else if (window?.location?.host === 'react-app-dev.cancer.gov') {
 	// This is for product testing
 	const dictSettings = {
 		...appParams,
 		...integrationTestOverrides,
 		...{ basePath: getProductTestBase() },
 	};
-	initialize(dictSettings);
+	window.DrugDictionaryApp(dictSettings);
```

Comment:
NIT: redundant to have 'window.DrugDictionaryApp(dictSettings);' in both if cases - can move outside since you're setting params in each.. (only diff is ...'{ basePath: getProductTestBase() },' )

Reply:
but we don't want the initialize call to happen automatically if not one of these two cases.